src/write_syslog.c

```diff
@@ -0,0 +1,697 @@
+/**
+ * collectd - src/write_syslog.c
+ * Copyright (C) 2012       Pierre-Yves Ritschard
+ * Copyright (C) 2011       Scott Sanders
+ * Copyright (C) 2009       Paul Sadauskas
+ * Copyright (C) 2009       Doug MacEachern
+ * Copyright (C) 2007-2012  Florian octo Forster
+ * Copyright (C) 2013-2014  Limelight Networks, Inc.
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation; only version 2 of the License is applicable.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin St, Fifth Floor, Boston, MA  02110-1301 USA
+ *
+ * Based on the write_graphite plugin. Authors:
+ *   Florian octo Forster <octo at collectd.org>
+ *   Doug MacEachern <dougm at hyperic.com>
+ *   Paul Sadauskas <psadauskas at gmail.com>
+ *   Scott Sanders <scott at jssjr.com>
+ *   Pierre-Yves Ritschard <pyr at spootnik.org>
+ * Based on the write_tsdb plugin. Authors:
+ *   Brett Hawn <bhawn at llnw.com>
+ *   Kevin Bowling <kbowling@llnw.com>
+ * write_syslog. Authors:
+ *   Shirly Radco <sradco@redhat.com>
+ **/
+
+/* write_syslog plugin configuation example
+ *
+ * <Plugin write_syslog>
+ *   <Node>
+ *     Host "localhost"
+ *     Port "44514"
+ *     Prefix "collectd"
+ *     MessageFormat "human"
+ *     HostTags ""
+ *   </Node>
+ * </Plugin>
+ *
+ *
+ * Host - Hostname or address to connect to. Defaults to localhost.
+ * Port - Service name or port number to connect to. Defaults to 44514.
+ * Prefix - Sets the metrics prefix string. Mandatory. Defaults to collectd.
+ * MessageFormat - Sets the format in which messages are sent to the broker,
```

Comment:
All this duplicates what's in the manual, and it will drift out of sync eventually. I think it's best reserved for collectd.conf(5)

Reply:
To remove from here completely?

Reply:
Is the collectd.conf.pod used for the collectd.conf(5) ?

Reply:
yes

Reply:
I don't see where I can put this information. Especially the format examples. And I dont know how to be able to see the collectd.conf.pod to make sure it looks ok...

Reply:
You've put the same information in the manpage. You can run pod2man on that file, or just make install and look at the installed manpage.

Reply:
Can you please give me the command to run for pod2man or instruct me where to look at the installed manpage? @rubenk

Reply:
fixed